setup.cfg

```
@@ -36,9 +36,10 @@ install_requires =
     tornado>=6.1.0
     jupyter_core
     jupyterlab_server~=2.10
-    jupyter_server~=1.4
+    jupyter_server>=1.13.4
```

Let's keep the upper-bound, for example?

```suggestion
    jupyter_server~=1.13
```

---

We can also switch to `jupyter_server>=1.13.4,<2` :+1: